### ISSUE TITLE
Set g:jsx_ext_required = 0 by default

### ISSUE THREAD

**Issue by mastertinner** [mxw/vim-jsx#158]
Using the `.jsx` file extension seems outdated and is not used by de facto standards like [create-react-app](https://github.com/facebook/create-react-app). To me it would therefore make sense to set `g:jsx_ext_required = 0` by default.

**Comment by trevordmiller** [comment]
I agree it would be great to set `let g:jsx_ext_required = 0` by default. Almost all JSX projects use `.js` files.
